src/requests/completions.jl

```diff
@@ -343,18 +343,24 @@ end
 
 function path_completion(doc, offset, rng, t, CIs)
     if t.kind == CSTParser.Tokenize.Tokens.STRING
-        path, partial = _splitdir(t.val[2:prevind(t.val, lastindex(t.val))])
-        if !startswith(path, "/")
-            doc_path = getpath(doc)
-            isempty(doc_path) && return
-            path = joinpath(_dirname(doc_path), path)
+        path = t.val[2:prevind(t.val, lastindex(t.val))]
+        if startswith(path, "~")
+            path = replace(path, '~' => homedir())
+            dir, partial = _splitdir(path)
+        else
+            dir, partial = _splitdir(path)
+            if !startswith(dir, "/")
```

Comment:
In general, does this work with Windows paths? Probably doesn't have to be addressed in this PR because it was present previously already.

Reply:
I don't think this works on Windows unless users starts vscode within git-bash, etc. Windows by default doesn't handle `~`, so it might be better to add the `startswith(path, "~")` only for unix systems.

Reply:
Oh, I was more thinking about the forward vs backslash situation :)

I think it is ok to expand `~` to `homedir` on Windows as well, for example in PowerShell that also works, I think.

Reply:
ah, I see. and yeah, I don't think `/` check doesn't work unless users type `foo/bar/baz.jl`. I remember `splitdir` works as expected with `/` even on Windows though.